### DISCUSSION TITLE
PDF printable documentation

### DISCUSSION THREAD

**Discussion by bashman** [quasarframework/quasar#10108] [General - Components / Directives / etc]
Is there any documentation in printable pdf format? 

I can build it?.

**Comment by pdanpdan** [comment]
no, there isn't, and it would be a huge waste of paper :)
what's wrong with the html documentation?

**Reply by bashman** [reply]
I do not need to print, only the print format. I wanted to put it on a kindle for offline reading. I'm pretty far away from the internet.

**Reply by pdanpdan** [reply]
The docs are pwa, load then once in a supporting browser and they will work without internet

**Reply by bashman** [reply]
> The docs are pwa, load then once in a supporting browser and they will work without internet

Yes, Cloning.
